### ISSUE TITLE
How to fix 302 found error?

### ISSUE THREAD

**Issue by nekozu** [Androz2091/insta.js#65]
(node:4) UnhandledPromiseRejectionWarning: IgResponseError: POST /api/v1/accounts/msisdn_header_bootstrap/ - 302 Found; 

I got this error and i dunno how to fix it. Can someone help me?

**Comment by Chrom-Faraon** [comment]
Yes i find way to fix it: go to node_modules then @androz2091/insta.js/src/structrures/Client.js then go to line 359 and delete it

**Comment by nekozu** [comment]
> Yes i find way to fix it: go to node_modules then @androz2091/insta.js/src/structrures/Client.js then go to line 359 and delete it

Thank you!

**Comment by farihdzkyy** [comment]
> Yes i find way to fix it: go to node_modules then @androz2091/insta.js/src/structrures/Client.js then go to line 359 and delete it

but if in heroku how? In my vps Is working but not in heroku

**Comment by Chrom-Faraon** [comment]
> > Yes i find way to fix it: go to node_modules then @androz2091/insta.js/src/structrures/Client.js then go to line 359 and delete it
> 
> but if in heroku how? In my vps Is working but not in heroku

just copy insta.js folder from node modules and put in your directory and replace require("@androz2091/insta.js") with require("./insta.js")

**Comment by Chrom-Faraon** [comment]
> > Yes i find way to fix it: go to node_modules then @androz2091/insta.js/src/structrures/Client.js then go to line 359 and delete it
> 
> Thank you!

You're welcome

**Comment by Androz2091** [comment]
Why is it working? It should not as you are not passing a state to the package right?
